src/charm.py

```diff
@@ -180,9 +181,27 @@ def _on_install(self, event: InstallEvent) -> None:
         if self.unit.is_leader():
             self.app_peer_data.update({"quorum": "default - non-ssl"})
 
-    def _on_zookeeper_pebble_ready(self, _: EventBase) -> None:
-        """Handler for the `upgrade-charm`, `zookeeper-pebble-ready` and `start` events."""
+    def _on_zookeeper_pebble_ready(self, event: PebbleReadyEvent) -> None:
```

Comment:
**todo**: `start` and `upgrade_charm` also point to this, so `EventBase` was better than `PebbleReadyEvent`.